cfme/infrastructure/virtual_machines.py

```diff
@@ -305,6 +308,19 @@ def is_displayed(self):
         )
 
 
+class VmTemplatesCompareView(TemplatesCompareView):
+    """Compare Templates page."""
+    title = Text('.//div[@id="center_div" or @id="main-content"]//h1')
+    comparison_table = Table(locator='//div[@id="compare-grid"]/table')
```

Comment:
Looks like the same widget, should be in the common view class.

Reply:
Updated as suggested.